runtime/composable/src/xcmp.rs

```diff
@@ -145,7 +154,7 @@ impl xcm_executor::Config for XcmConfig {
 	type AssetTransactor = LocalAssetTransactor;
 
 	type OriginConverter = XcmOriginToTransactDispatchOrigin;
-	type IsReserve = MultiNativeAsset;
+	type IsReserve = MultiNativeAsset<AbsoluteReserveProvider>;
```

Comment:
may you run `cargo make run-local-integration-tests-debug` and wait it pass. promise will do it part of CI in next XCMP PR